README.md

```diff
@@ -1885,6 +1910,122 @@ If you find yourself using the `only` modifier often, you can set the [context_b
 
 Components can also access the outer context in their context methods like `get_context_data` by accessing the property `self.outer_context`.
 
+## Customizing component tags with TagFormatter
+
+_New in version 0.89_
+
+By default, components are rendered using the pair of `{% component %}` / `{% endcomponent %}` template tags:
+
+```django
+{% component "button" href="..." disabled %}
+Click me!
+{% endcomponent %}
+
+{# or #}
+
+{% component "button" href="..." disabled / %}
+```
+
+You can change this behaviour in the settings under the [`COMPONENTS.tag_formatter`](#tag-formatter-setting).
+
+For example, if you set the tag formatter to `django_components.shorthand_component_formatter`, the components will use their name as the template tags:
+
+```django
+{% button href="..." disabled %}
+  Click me!
+{% endbutton %}
+
+{# or #}
+
+{% button href="..." disabled / %}
+```
+
+### Available TagFormatters
+
+django_components provides following predefined TagFormatters:
+
+- **`ComponentFormatter` (`django_components.component_formatter`)**
+
+- **`ShorthandComponentFormatter` (`django_components.shorthand_component_formatter`)**
```

Comment:
Short description?

Reply:
Added in https://github.com/EmilStenstrom/django-components/pull/572/commits/9061c3226ae70e2a23788063c3127d4136059752